### ISSUE TITLE
Everything completed except for last step (TFT)

### ISSUE THREAD

**Issue by benghee** [Blackymas/NSPanel_HA_Blueprint#925]
I just flashed the Nspanel successfully but when it comes to the last step for upload the tft file, nothing happens. I then checked the esphome add on, in the code that one mistake i made was the ip of home assistant was incorrect. After i changed it to the right IP that can download the tft file to my computer, nothing happen too when i click the upload button in configuration. 

Now should i start all over the flash process again because i input the wrong IP in the code? or there is something else i can fix. By the way, i already tried all the solutions in Common Issues TFT Upload section.

**Comment by rleongcs** [comment]
Yes...you need to correct the ESPHome code to have the right IP and wirelessly update the code as that would be used as the URL to grab the TFT file.

**Comment by edwardtfn** [comment]
@benghee, any update to this? Should we close this issue?
